docs/content/docs/templates.md

```diff
@@ -575,6 +575,19 @@ Example:
 ### iterable
 Returns true if the given variable can be iterated over in Tera (ie is an array/tuple).
 
+### startswith
+Returns true if the given variable is a string starts with the arg given.
+
+Example:
+```jinja2
+{% if path is startswith "x/" %}
```

Comment:
0.11 require parentheses around tests params so that should be `{% if path is startswith("x/") %}

Reply:
Ah, ok. Probably it makes sense to have a doc test for these documents as well :)